bokeh/models/tiles.py

```diff
@@ -26,11 +26,11 @@ class TileSource(PlotObject):
     the maximum zoom level for the tile layer. This is the most "zoomed-in" level.
     """)
 
-    x_origin_offset = Float(default=20037508.34, help="""
+    x_origin_offset = Float(default=None, help="""
```

Comment:
Properties don't have this mechanism right now - there's no support for an "abstract property" which is further refined in a subclass. If the property is different in two subclasses, it has to go in the subclasses. For the most part the property in the base class is simply ignored and does nothing, but to the extent it's ever used, it can only cause bugs and/or waste memory. To fix that I think we'd need to tweak properties.py to have a thing like `initial_resolution = Abstract(Float())` and then properties.py would validate that the subclasses conform to the abstract property ... or maybe some other solution.

In the interest of fixing this quickly, here's a PR which applies to this PR, showing what I'd do: https://github.com/bokeh/bokeh/pull/3141
If it looks good in theory you may want to test it since you'll better be able to tell if the examples are correct.